### PR TITLE
Fix/dask cluster

### PR DESCRIPTION
**Pull request by FariborzDaneshvar-NOAA**
Moved dask cluster from main to cli in analyze_ensemble.py

### CODE REVIEW

File: stormworkflow/post/analyze_ensemble.py [hunk 1]
```diff
@@ -408,19 +408,19 @@ def cli():
...
                            memory="500GB",
-                           account="compute",
+                           account="nos-surge", #PW:"compute" ; Hercules:"nos-surge" or "nosofs"
                            walltime="08:00:00",
-                           header_skip=['--mem'],
-                           interface="eth0") 
-    cluster.scale(6) 
-    client = Client(cluster) 
+                           header_skip=['--mem']) #,
+#                           interface="eth0")     # only for PW
```
Comment (SorooshMani-NOAA) [inline stormworkflow/post/analyze_ensemble.py:417]: For future reference, it's better to remove these platform specific hardcoded values and get the values from the `input.yaml` file, similar to what we do for HPC account and solver ntasks:

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/refs/input.yaml#L18-L21

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/scripts/workflow.sh#L42-L47

https://github.com/oceanmodeling/ondemand-storm-workflow/blob/13103fa920aa7be8d993f2408237d57b74a7f676/stormworkflow/scripts/workflow.sh#L168-L173

Comment (SorooshMani-NOAA) [reply stormworkflow/post/analyze_ensemble.py:417]: For now I'll merge this to fix the issue with the location of cluster creation within the script

Comment (SorooshMani-NOAA) [reply stormworkflow/post/analyze_ensemble.py:417]: @FariborzDaneshvar-NOAA when you have time, can you please create another ticket about removal of those hardcoded values?

Comment (FariborzDaneshvar-NOAA) [reply stormworkflow/post/analyze_ensemble.py:417]: Sure, I will.

**Pull request merged**